src/api.rs

```diff
@@ -310,6 +312,7 @@ impl ChromaSampling {
   }
 }
 
+/// Contains details of chroma sampling position
```

Comment:
"details" is a bit loose, it provide 1 detail, that is the location of chroma wrt luma samples, a brief explanation like below would be nice